lib/cylc/config.py

```diff
@@ -1456,9 +1457,10 @@ def generate_taskdefs(self, orig_expr, left_nodes, right, seq):
             try:
                 name, offset_is_from_icp, _, offset, _ = (
                     GraphNodeParser.get_inst().parse(node))
-            except GraphNodeError, x:
+            except GraphNodeError as exc:
+>>>>>>> Removed unused imports; use _ for unused vars (Pylint).
```

Comment:
Ah, man.  I tried to do the world's fastest conflict resolution on the way out of the office.  Spectacular fail.